Formula/okta-aws-cli.rb

```diff
@@ -0,0 +1,18 @@
+class OktaAwsCli < Formula
+  desc "Okta authentication in support of aws cli operation"
+  homepage "https://github.com/okta/okta-aws-cli"
+  url "https://github.com/okta/okta-aws-cli/archive/refs/tags/v0.2.1.tar.gz"
+  sha256 "06374f0eb3e371a8ef1a5e8fb2bcd0e5bfcac607aa2083f6c1101b54713a47bf"
+  license "Apache-2.0"
+
+  depends_on "go"
+  def install
+    system "go", "build", "-o", bin/"okta-aws-cli", "cmd/okta-aws-cli/main.go"
+  end
+
+  test do
+    str_help = shell_output("#{bin}/okta-aws-cli --help")
```

Comment:
We need a test that exercises the some of the functionality of the app. Version checks or usage checks (foo --version or foo --help) are not sufficient, as explained in the formula cookbook.

In most cases, a good test would involve running a simple test case: run #{bin}/foo input.txt.

- Then you can check that the output is as expected (with assert_equal or assert_match on the output of shell_output)
- You can also check that an output file was created, if that is expected: assert_predicate testpath/"output.txt", :exist?

Some advice for specific cases:

- If the formula is a library, compile and run some simple code that links against it. It could be taken from upstream's documentation / source examples.
- If the formula is for a GUI program, try to find some function that runs as command-line only, like a format conversion, reading or displaying a config file, etc.
- If the software cannot function without credentials, a test could be to try to connect with invalid credentials (or without credentials) and confirm that it fails as expected.
- Same if the software requires a virtual machine, docker instance, etc. to be running.

Reply:
New test added

Reply:
@SMillerDev New test added